### PR TITLE
Add javascript and fixes

### PR DESCRIPTION
**Pull request by Matt-1985**
Add Javascript and general fixes like styling on the track

### CODE REVIEW

**Review by Sina-Sche — APPROVED** [review]
good job 👍

File: src/components/Track/track.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.track {
+  display: grid;
+  align-items: center;
+  grid-template: auto / auto 1fr auto;
```
Comment (BelaM80) [inline src/components/Track/track.css:4]: Hey, good job! Just one thing: grid-template goes both directions, vertical and horizontal. You only have three object in one column, so you could use grid-template-columns: auto 1fr auto;.

**Review by BelaM80 — APPROVED** [review]

**Review by boldCM — APPROVED** [review]
lgtm 👍

File: src/components/Track/track.css (hunk 1 above)
Comment (Matt-1985) [reply src/components/Track/track.css:4]: i'll have a check on that 👍🏻  thank you Bela

**Pull request merged**